File: progra3-0583-equipo5-frontend/Frontend_C#/SoftPac/SoftPacBusiness/MonedasBO.cs
using SoftPac.Model;
using SoftPac.Persistance.DAO;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoftPac.Business
{
    public class MonedasBO
    {
        private MonedasDAO monedasDAO;

        public MonedasBO()
        {
            monedasDAO = new MonedasDAOImpl();
        }
        public BindingList<MonedasDTO> ListarTodos()
        {
            return monedasDAO.ListarTodos() as BindingList<MonedasDTO>;
        }
        public MonedasDTO ObtenerPorId(int monedaId)
        {
            return monedasDAO.ObtenerPorId(monedaId);
        }

    }
}

File: BackEnd/Backend_Java/SoftPacBusiness/src/main/java/pe/edu/pucp/softpac/bo/MonedasBO.java

package pe.edu.pucp.softpac.bo;

import java.util.ArrayList;
import pe.edu.pucp.softpac.dao.MonedasDAO;
import pe.edu.pucp.softpac.daoImpl.MonedasDAOImpl;
import pe.edu.pucp.softpac.model.MonedasDTO;

public class MonedasBO {
    
    private MonedasDAO monedasDAO;
    
    public MonedasBO(){
        monedasDAO = new MonedasDAOImpl();
    }
    
    public ArrayList<MonedasDTO> listarTodos(){
        return (ArrayList<MonedasDTO>) this.monedasDAO.listarTodos();
    }
    
    public MonedasDTO obtenerPorID(Integer moneda_id){
        MonedasDTO monedasDTO = new MonedasDTO();
        monedasDTO.setMoneda_id(moneda_id);
        return this.monedasDAO.obtenerPorId(moneda_id);
    }
    
    
}
